src/components/qualityOptions.js

```diff
@@ -53,7 +53,7 @@ export function getVideoQualityOptions(options) {
 
     if (maxStreamingBitrate) {
         let selectedIndex = -1;
-        for (let i = 0, length = qualityOptions.length; i < length; i++) {
+        for (let i = 1, length = qualityOptions.length; i < length; i++) {
```

Comment:
`auto` is optional, so sometimes `i` must start from `0`.
https://github.com/jellyfin/jellyfin-web/blob/aadd99e617a48ac11e128e8ac0f63d48d42f43cd/src/components/qualityOptions.js#L24-L26